components/header-search-box/header-search-box.tsx

```diff
@@ -26,98 +26,123 @@ const HeaderSearchBox = ({ placeholderText }: { placeholderText: string }): Reac
-          opacity: 1; /* Firefox */
-        }
+          .ais-SearchBox .ais-SearchBox-input::placeholder {
+            color: ${theme.colors.silver};
+            font-style: italic;
+            opacity: 1; /* Firefox */
+          }
 
-        .ais-SearchBox-input:-ms-input-placeholder {
-          color: ${theme.colors.silver};
-        }
+          .ais-SearchBox-input:-ms-input-placeholder {
+            color: ${theme.colors.silver};
+          }
 
-        .ais-SearchBox-input::-ms-input-placeholder {
-          color: ${theme.colors.silver};
-        }
+          .ais-SearchBox-input::-ms-input-placeholder {
+            color: ${theme.colors.silver};
+          }
 
-        .ais-SearchBox-input:focus {
-          background-color: #fff;
-          color: ${theme.colors.doveGray};
-          border: 1px solid #039b35;
-        }
+          .ais-SearchBox-input:focus {
+            background-color: #fff;
+            color: ${theme.colors.doveGray};
+            border: 1px solid #039b35;
+          }
 
-        .ais-SearchBox-submit {
-          align-items: center;
-          display: flex;
-          position: absolute;
-          justify-content: center;
-          transform: translateY(-50%);
-          left: 0;
-          top: 50%;
-          height: 100%;
-          color: ${theme.colors.silver};
-          width: 64px;
-        }
+          .ais-SearchBox-submit {
+            align-items: center;
+            display: flex;
+            position: absolute;
+            justify-content: center;
+            transform: translateY(-50%);
+            left: 0;
+            top: 50%;
+            height: 100%;
+            color: ${theme.colors.silver};
+            width: 64px;
+          }
 
-        .ais-SearchBox-reset {
-          right: 0;
-          color: ${theme.colors.silver};
-        }
+          .ais-SearchBox-reset {
+            right: 0;
+            color: ${theme.colors.silver};
+          }
+
+          .ais-SearchBox-reset,
+          .ais-SearchBox-loadingIndicator,
+          .ais-SearchBox-submit {
+            align-items: center;
+            -webkit-appearance: none;
+            -moz-appearance: none;
+            appearance: none;
+            display: flex;
+            height: 100%;
+            justify-content: center;
+            position: absolute;
+            top: 50%;
+            transform: translateY(-50%);
+            width: 48px;
+            border: 0;
+            background: none;
+          }
 
-        .ais-SearchBox-reset, .ais-SearchBox-loadingIndicator, .ais-SearchBox-submit {
-          align-items: center;
-          -webkit-appearance: none;
-          -moz-appearance: none;
-          appearance: none;
-          display: flex;
-          height: 100%;
-          justify-content: center;
-          position: absolute;
-          top: 50%;
-          transform: translateY(-50%);
-          width: 48px;
-          border: 0;
-          background: none;
```

Comment:
This css was missing a closing curly bracket.  Fixing this has caused prettier to fix some white spacing issues seen above.